packages/eslint-config-godaddy-minimal/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy-react-flow')
+  ],
+  rules: {
+    'complexity': 0,
+    'consistent-return': 0,
+    'id-length': 0,
+    'indent': [2, 2, {
+      VariableDeclarator: { var: 2, let: 2, const: 3 },
+      flatTernaryExpressions: true,
+      MemberExpression: 'off'
+    }],
+    'max-params': 0,
+    'max-nested-callbacks': 0,
+    'max-statements': 0,
+    'no-else-return': 0,
+    'no-nested-ternary': 0,
+    'no-shadow': 0,
```

Comment:
I'm on the fence for this one. Accidental shadowing can cause huge headaches, and may be easily missed during code reviews. It is an annoyance during the scenario described above. I'm +1 downgrading this to a warning.

Reply:
If accidental shadowing is possible, it's a sign that the scope was too large and/or the names being used were too vague, which should both be discussed in code review on their own.

Reply:
Eh, yeah I see that. Shadowing is harder to catch in code reviews since the original declaration might not be visible in the GitHub reviewer (they'd have to click to expand upwards and/or downwards), in the case where existing code is being modified.

But, in keeping that this is a new "minimal" style guide, I'm OK with this. There will be potential problems either way.